helper/include/odp/helper/cli.h

```diff
@@ -33,6 +33,9 @@ extern "C" {
  * @{
  */
 
+/** User defined command function type. */
+typedef void (*odph_cli_user_cmd_func_t)(int argc, char *argv[]);
```

Comment:
In addition to the function pointer a user could also pass the CLI command sting.

Reply:
The idea here is that the application can implement whatever command hierarchy it wants with argv[]. The user function can for example treat argv[0] as the name of a command. In comparison to this, having multiple functions and strings would just impose unnecessary limitations.

Reply:
@psavol pointed out in a discussion that independent commands are likely needed by for example EM and application. I'll add support for multiple independently registered user commands.

Reply:
Jere, if I understand your motivation correctly that you are trying to add an extension to the existing CLI framework? In form of one function and one ("user") command? If existing CLI is missing such capabilities -- to add user command(s)? If so, then, it is probably better (more generic and practical) to make such extension by adding a table of commands (instead of one command) or add multiple commands (one by one) to the command list.

Reply:
Changed in v2.